gitlint/tests/test_body_rules.py

```diff
@@ -111,6 +117,12 @@ def test_body_min_length(self):
         violations = rule.validate(commit)
         self.assertIsNone(violations)
 
+        # Skip body checks, if regex matches
+        rules.BodyMinLength({'ignore-body-checks-regex': 'Release\s([a-z\-])*\s(\d+\.)?(\d+\.)?(\*|\d+)'})
+        commit = self.gitcommit(u"Release foo-bar-baz 0.12.3")
+        violations = rule.validate(commit)
+        self.assertIsNone(violations)
+
```

Comment:
That test should go into a separate test method.

Reply:
See above